airbyte-workers/src/main/java/io/airbyte/workers/WorkerConstants.java

```diff
@@ -29,7 +29,8 @@
   public static final String TAP_CONFIG_JSON_FILENAME = "tap_config.json";
   public static final String TARGET_CONFIG_JSON_FILENAME = "target_config.json";
 
-  public static final String CATALOG_JSON_FILENAME = "catalog.json";
+  public static final String TAP_CATALOG_JSON_FILENAME = "tap_catalog.json";
```

Comment:
should we rename these constants to 
- source_catalog.json / destination_catalog.json?
- source_config.json / destination_config.json?

Reply:
Sounds good to me!

Reply:
If you do so, also rename the const names

Reply:
agreed